### PR TITLE
Add classes

### PR DESCRIPTION
**Pull request by AnselemOdims**
# Steps Taken for this Milestone
- Created a Person class
- Initialized the constructor with instance variables
- Created getters and setters for the variables
- Created a private method is_of_age?
- Created a public method can_use_services?
- Created a Student class that inherits from the Person class
- Ensured the child constructor extended from the parent constructor by adding @classroom and a parameter for it.
- Added a method play_hooky that returns "¯\(ツ)/¯"
- Created a Teacher class that inherits from the Person class
- Ensured the child constructor extended from the parent constructor by adding @specialization and a parameter for it.
- Overrode the can_use_services? method so it always returns true

### CODE REVIEW

File: person.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..500)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    is_of_age? || @parent_permission
```
Comment (SajjadAhmad14) [inline person.rb]: Please use method `of_age?` here as `is_of_age?` is not defined in the file.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Anselem, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
README is professional. :heavy_check_mark:
Git flow is followed. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
**_Note: Please do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: person.rb (hunk 1 above)
Comment (AnselemOdims) [reply person.rb]: Thanks @SajjadAhmad14 but `rubocop` keeps flagging it as an offence and said it should be changed to `of_age`

Comment (SajjadAhmad14) [reply person.rb]: Hi @AnselemOdims so you need to use `#of_age?` inside `can_use_services?`.

Comment (AnselemOdims) [reply person.rb]: Thanks @SajjadAhmad14

**Review by dasileker — APPROVED** [review]
### **Hi @AnselemOdims,** 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**